config.yml

```diff
@@ -0,0 +1,7 @@
+api_version: 1.1.0
+stac_version: 0.9.0
+url: http://www.brockmann-consult.de/xcube-geoDB-openEO
```

Comment:
Such information should go into the code base as a resource. The reason is, that this is not strictly configuration but instead is bound to the current implementation.

Reply:
Done.